app/controllers/tickets_controller.rb

```diff
@@ -1,48 +1,57 @@
 class TicketsController < ApplicationController
-    def index
-        render json: @tickets
-    end
+  def index
+    @tickets = Tickets::UseCases::FetchWithColumns.new.call(params[:reservation_id])
+    render json: Tickets::Representer.new(@tickets).basic
+  end
+
+  def show
+    ticket = Tickets::Repository.new.find_by(params[:id])
+    render json: Tickets::Representer.new([ticket]).extended
+  end
 
-    def show
-        set_ticket
-        render json: @ticket
+  def create
+    if Tickets::UseCases::SeatAvailable.new(ticket_params[:reservation_id],ticket_params[:seat]).call
+      ticket = Tickets::UseCases::Create.new.call(params: ticket_params)
+      
+      if ticket.valid?
+          update_screening_seats
+          render json: ticket, status: :created
+      else
+          render json: ticket.errors, status: :unprocessable_entity
+      end
+    else
+      render json: {status: "seat taken"}
     end
+  end
```

Comment:
```suggestion
  def create
    return render json: {status: "seat taken"} unless Tickets::UseCases::SeatAvailable.new(ticket_params[:reservation_id],ticket_params[:seat]).call
 
    ticket = Tickets::UseCases::Create.new.call(params: ticket_params)   
    return render json: ticket.errors, status: :unprocessable_entity unless ticket.valid?
    
    update_screening_seats
    render json: ticket, status: :created
  end
```

It may be just my preference, but I would say having it more like that is easier to understand. That way every case - seat taken, invalid data and creating ticket is always read from top to bottom.